sdk/storage/azure-storage-blob/azure/storage/blob/_upload_helpers.py

```diff
@@ -316,12 +326,12 @@ def upload_append_blob(  # pylint: disable=unused-argument
                 append_position_access_conditions=append_conditions,
                 progress_hook=progress_hook,
                 headers=headers,
-                **kwargs)
+                **kwargs))
         except HttpResponseError as error:
-            if error.response.status_code != 404:
+            if error.response.status_code != 404:  # type: ignore
```

Comment:
`azure\storage\blob\_upload_helpers.py:331: error: Item "None" of "Optional[_HttpResponseCommonAPI]" has no attribute "status_code"  [union-attr]`